wxt_cidrbot/git_api_handler.py

```diff
@@ -84,14 +84,16 @@ def get_issue_info(self, issue, issue_type):
 
     def get_assigned_status(self, assigned_user, issue_type, request):
         text = ""
+        assigned_status = []
         self.logging.debug(issue_type)
         if assigned_user is not None:
             text += f" | **Assigned**: " + str(assigned_user) + "\n"
-            assigned_status = "True" + ", " + str(assigned_user)
+            assigned_status.append(True)
```

Comment:
You could rewrite this as:
```
assigned_status = (True, str(assigned_user))
``` 
(likewise with the false case).  Then you don't need to initialize an emptry array, and it might be more clear what you're doing.